api/src/main/resources/liquibase-schema-only.xml

```diff
@@ -1828,6 +1828,7 @@
             </column>
             <column name="username" type="varchar(50)"/>
             <column name="password" type="varchar(128)"/>
+            <column name="email" type="varchar(255)"/>
```

Comment:
This should have a unique constraint

Reply:
I know for a fact that MySQL allows a unique constraint on a nullable field, the problem might be that since in unit tests we use h2 database it might not support it, you can confirm it by running in tests in MySQL mode

Reply:
This one had no issues it the not null constraint you said i add on the file api/src/main/resources/org/openmrs/api/db/hibernate/User.hbm.xml